docs/getting-started.md

```diff
@@ -22,7 +22,7 @@ Install the
 package:
 
 ```sh
-dotnet add package OpenTelemetry.Exporter.Console -v 0.2.0-alpha.419
+dotnet add package OpenTelemetry.Exporter.Console -v 0.3.0-beta.1
```

Comment:
i think we should not set the version. so it would get the latest

Reply:
done.